packages/nx/src/daemon/server/server.ts

```diff
@@ -244,6 +251,8 @@ async function handleMessage(socket, data: string) {
     );
   } else if (payload.type === 'REGISTER_FILE_WATCHER') {
     registeredFileWatcherSockets.push({ socket, config: payload.config });
+  } else if (payload.type === REGISTER_PROJECT_GRAPH_LISTENER) {
```

Comment:
Lets add these messages to message-types, gives a bit more type safety but is admittedly a bit more cruft